tun/tun.go

```diff
@@ -52,10 +52,14 @@ type Device interface {
 	BatchSize() int
 }
 
-type LinuxDevice interface {
+// GRODevice is a Device extended with methods for disabling GRO. Certain OS
+// versions may have offload bugs. Where these bugs negatively impact throughput
+// or break connectivity entirely we can use these methods to disable the
+// related offload.
```

Comment:
wouldn't hurt to add a paragraph to this godoc with some more details (or a URL to more details) about the known problem(s) and their commit ranges/versions.

For upstreaming, a paragraph without a Tailscale URL is probably best?

Reply:
(The commit message is great, btw. Maybe just mirror that, but also include Linux kernel versions, if known? As LTS complicates it.)